### ISSUE TITLE
Не одно, а два слова.

### ISSUE THREAD

**Issue by Nennner** [olegcherr/Reedy-for-Chrome#19]
Олег, почему бы не добавить возможность настраивать количество слов (к примеру показывается не одно, а несколько слов)? И у пользователя есть возможность выбирать сколько именно.
И количество строчек. Безусловно, придётся водить глазами, но тем не менее, кому-нибудь да нравиться фиксировать сразу "много текста" глазами.
Очень хороша ночная тема, очень хороша.
Я не знаю точно, но может стоит добавить настраиваемую задержку после запятой/точки? И больше настроек (безусловно для тех, кто захочет покапаться спрятать их...да хоть в правую кнопку на иконку в "Настройки".
Очень хорошо, что время оставшееся показывает!
Великолепный проект!
Решите, стоит ли внедрять или нет, нужно ли оно, удобно ли.
dasfadsfdas.asdfadsfas@yandex.ru

**Comment by amal** [comment]
> Безусловно, придётся водить глазами

Поэтому и нет смысла добавлять такое. Как только человек начинает делать саккады (водит глазами) скорость чтения падает. Если времени слишком много и можно водить глазами, вероятнее всего стоит просто увеличивать скорость.

А вообще, уже сейчас же есть функция отображения продолжения текста. Она отлично сочетается с RSVP, позволяя сохранять фокусировку в одной точке.

**Comment by cfytrok** [comment]
> Безусловно, придётся водить глазами

Не согласен. Во всех учебниках по скорочтению есть этап, когда задача стоит читать строчку за 2-3 фиксации взгляда. Воспринять за одну фиксацию 2-3 слова без саккад не проблема, если делал упражнения на расширения поля зрения. Мне бы хотелось увидеть эту функцию в расширение. Но в реализации возникнет несколько проблем: куда ставить фокус и как группировать слова для показа (думаю количество слов должно меняться динамически). Насколько я понимаю, когда читаешь 2-3 фиксациями мозг воспринимает текст из парафовеальной области и направляет взгляд для следующей фиксации в центр смыслового блока (например прилагательное+существительное).

Кстати про функцию продолжения текста. Добавьте, пожалуйста, аналогичную функцию начала текста. В моем представлении аппарат чтения устроен как конвеер. Пока глаза видят третье слово в предложении, он узнает второе, и понимает смысл первого. Мне кажется в момент показа третьего "кадра" для уточнения смысла слова из первого "кадра" мозг может использовать информацию из парафовеальной области.

В текущей версии главные фишки расширения - это помощь в избавлении от артикуляции и привычки читать медленно. После использования начинаешь по-другому читать тексты без расширения.
